src/bootstrap/src/core/config/config.rs

```diff
@@ -1869,7 +1864,9 @@ impl Config {
                         }
                     }
 
-                    if let Some(config_path) = &self.config {
+                    // Ferrocene addition: our CI builders don't currently upload builder-config.
+                    // For now, just assume it matches the local settings.
+                    if false && let Some(config_path) = &self.config {
```

Comment:
maybe left-over from testing
```suggestion
                    if let Some(config_path) = &self.config {
```

Reply:
I didn't remove the code altogether because I don't want to cause merge conflicts with upstream.